util/file.go

```diff
@@ -83,6 +84,7 @@ func GetPathRelativeTo(path string, basePath string) (string, error) {
 		return "", errors.WithStackTrace(err)
 	}
 
+	relPath = strings.Replace(relPath, `\`,"/", -1)
```

Comment:
Additionally, pathRelativeToInclude can actually call findInParentFolders indirectly through [ResolveTerragruntConfigString](https://github.com/gruntwork-io/terragrunt/blob/master/config/config_helpers.go#L83). So simple escaping is actually double escaping in many scenarios.